db/migrate/20220817124428_add_email_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToUser < ActiveRecord::Migration[7.0]
```

Comment:
Inconsistency in naming the file
`user` or `users`?

Reply:
Fixed